reference/technical_vision/1.5/architecture.rst

```diff
@@ -0,0 +1,127 @@
+│       ├── EntityConfigBundle      -> [Done] - removed (ServiceLinkPass has been extracted in Oro/SecurityBundle)
+│       ├── DistributionBundle      -> [Done] - removed (automatic routing has been dropped and routes are explicitly declared in routing.yml)
+│       ├── FilterBundle            -> -
+│       ├── FormBundle              -> merge useful parts to Oro/ConfigBundle and Pim/EnrichBundle
+│       ├── LocaleBundle            -> [Done] - removed
+│       ├── NavigationBundle        -> -
+│       ├── RequireJSBundle         -> -
+│       ├── SecurityBundle          -> -
+│       ├── TranslationBundle       -> merge useful parts to Akeneo/Pim Localization then remove
+│       ├── UIBundle                -> merge useful parts to Pim/UIBundle
+│       └── UserBundle              -> merge useful parts to Pim/UserBundle
+└── Pim
+    ├── Bundle
+    │   ├── AnalyticsBundle         -> -
+    │   ├── BaseConnectorBundle     -> could be totally deprecated (but kept with tests) once exports re-worked in ConnectorBundle
+    │   ├── CatalogBundle           -> continue to extract business code to relevant components
+    │   ├── CommentBundle           -> could be splitted in a Akeneo component + bundle (does not rely on PIM domain)
+    │   ├── ConnectorBundle         -> could welcome new classes if we re-work export
+    │   ├── DashboardBundle         -> -
+    │   ├── DataGridBundle          -> move generic classes to Oro/DataGridBundle, move specific related to product to Pim/EnrichBundle
+    │   ├── EnrichBundle            -> could contain all Akeneo PIM UI (except independent bundles as workflow, pam)
+    │   ├── FilterBundle            -> merge in Oro/DataGridBundle or Pim/DataGridBundle
+    │   ├── ImportExportBundle      -> could be merged to EnrichBundle it mainly contain UI related classes
+    │   ├── InstallerBundle         -> -
+    │   ├── JsFormValidationBundle  -> -
+    │   ├── NavigationBundle        -> -
+    │   ├── NotificationBundle      -> bit re-worked during the collaborative workflow epic
+    │   ├── PdfGeneratorBundle      -> -
+    │   ├── ReferenceDataBundle     -> -
+    │   ├── TransformBundle         -> move normalizer/denormalizer part and deprecate all other parts (related to deprecated import system)
+    │   ├── TranslationBundle       -> copy useful classes in new Localization component + bundle, then remove this bundle
+    │   ├── UIBundle                -> mainly used for js/css third party libraries, we should load them via a dedicated package manager
+    │   ├── UserBundle              -> merge used parts of Oro/UserBundle to Pim/UserBundle
+    │   ├── VersioningBundle        -> -
+    │   └── WebServiceBundle        -> -
+    └── Component
+        ├── Catalog
+        ├── Connector
+        └── ReferenceData
+```
+
+Ideally, the 1.5 version could be the following, (depending on the amount of tech cleaning we manage to do),
+
+```
+src/
+├── Acme
```

Comment:
I think this kind of documentation about what is the purpose of each bundle is really missing in tech doc. However, it's not the good place for it here. We should put it elsewhere.